pkg/ir/ir_generator.go

```diff
@@ -492,11 +490,11 @@ func inferStringValue(expr string) interface{} {
 func parseFeatureColumn(el *parser.ExprList) (FeatureColumn, error) {
 	head := (*el)[0].Value
 	if head == "" {
-		return nil, fmt.Errorf("column description expects format like NUMERIC(key) etc, got %v", el)
+		return nil, fmt.Errorf("column description expects format like DENSE(key) etc, got %v", el)
 	}
 
 	switch strings.ToUpper(head) {
-	case numeric:
+	case dense, sparse:
```

Comment:
Can sparse tensor used in numeric columns?

Reply:
Sparse tensor is not allowed in TensorFlow numeric columns, but it would work in XGBoost numeric columns defined by ourselves.

Reply:
I see.